packages/providers/upload-aws-s3/README.md

```diff
@@ -30,6 +30,7 @@ npm install @strapi/provider-upload-aws-s3 --save
   - `signedUrlExpires` is the number of seconds before a signed URL expires. (See [how signed URLs work](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-signed-urls.html)). Defaults to 15 minutes and URLs are only signed when ACL is set to `private`.
   - `Bucket` is the name of the bucket to upload to.
 - `actionOptions` is passed directly to the parameters to each method respectively. You can find the complete list of [upload/ uploadStream options](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property) and [delete options](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#deleteObject-property)
+- `buildFileUrl` optionally pass this function to define custom logic for building file url. This is provided as a way to handle the many different s3 provider options.
```

Comment:
I was thinking if this name is expressive enough to understand what  is doing.

At first it might mean multiple things,
- Build url before being uploaded
- Build url for url signing
- Build url after it has been upload

wdyt if we find alternative names that make this difference more clear? A name that conveys it decodes the uploaded file data into the url that will be saved into db?

Reply:
Yeah good point. How about `mapFileDataToDbUrl` ?

Reply:
That definetly sounds bettter, I guess `mapFileDataToUrl`  without "Db" would be expressive enough!

I wonder if adding something like:
```
config: {
   ...
   mappers: {
     fileDataToUrl: () => {}
    }
 }
 ```
 
 would be more extensible or if its just too much